### PR TITLE
Adding Triton notebooks

### PR DESCRIPTION
**Pull request by gvashishtha**
Wasn't able to run black-nb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,34 +25,35 @@ The main example notebooks are located in the [notebooks directory](notebooks).
...
+[notebooks/tensorflow-v2/train-mnist-nn.ipynb](notebooks/tensorflow-v2/train-mnist-nn.ipynb)|AML - GPU|tensorflow2, mlflow|mnist|conda file|None|None
 [notebooks/lightgbm/train-iris.ipynb](notebooks/lightgbm/train-iris.ipynb)|AML - CPU|lightgbm, mlflow|iris|pip file|None|None
-[notebooks/pytorch/train-mnist-cnn.ipynb](notebooks/pytorch/train-mnist-cnn.ipynb)|AML - GPU|pytorch, mlflow|mnist|conda file|None|None
 [notebooks/pytorch/train-mnist-mlproject.ipynb](notebooks/pytorch/train-mnist-mlproject.ipynb)|AML - GPU|pytorch, mlflow|mnist|conda file|None|mlproject
+[notebooks/pytorch/train-mnist-cnn.ipynb](notebooks/pytorch/train-mnist-cnn.ipynb)|AML - GPU|pytorch, mlflow|mnist|conda file|None|None
+[notebooks/xgboost/train-iris.ipynb](notebooks/xgboost/train-iris.ipynb)|AML - CPU|xgboost, mlflow|iris|pip file|None|None
+[notebooks/sklearn/train-diabetes-mlproject.ipynb](notebooks/sklearn/train-diabetes-mlproject.ipynb)|AML - CPU|sklearn, mlflow|diabetes|conda file|None|None
+[notebooks/sklearn/train-diabetes-ridge.ipynb](notebooks/sklearn/train-diabetes-ridge.ipynb)|AML - CPU|sklearn, mlflow|diabetes|conda file|None|None
```
Comment (lostmygithubaccount) [inline README.md:39]: should probably adjust `readme.py` to result in consistent ordering

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 2]
```diff
@@ -0,0 +1,268 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Copyright (c) Microsoft Corporation. All rights reserved.\n",
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:7]: this isn't needed - follow style of other notebooks

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:33]: remove this cell

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+    "\n",
+    "print(azureml.core.VERSION)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Get workspace\n",
+    "Load existing workspace from the config file info. If you are running this notebook in a Compute Instance, a configuration file has already been created for you. If you are running this notebook somewhere else, please follow the steps to [create a configuration file](https://docs.microsoft.com/azure/machine-learning/how-to-configure-environment#workspace)."
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:44]: see other notebooks

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "\n",
+    "!python setup.py --compute_loc='westus2' --vm_size='Standard_NC6'"
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:95]: can we do this in python code in the cell? not a fan of running a bunch of scripts from jupyter cells

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 6]
```diff
@@ -0,0 +1,268 @@
...
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "from azureml.core.model import Model\n",
+    "import os\n",
+    "\n",
+    "model = Model.register(\n",
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:119]: see other notebooks - follow convention of using Pathlib and prefix.joinpath

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy the model as a web service\n",
+    "First create a scoring script. You can see the one we created for you in the `scripts` directory. Then, create an InferenceConfig and a DeploymentConfig and call Model.deploy().\n",
+    "\n",
+    "Note that this step may take 10-15 minutes to run. "
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:137]: all notebooks need to run in <10 minutes, how can we speed this one up?

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+    "\n",
+    "inference_config = InferenceConfig(\n",
+    "    # This entry script is where we dispatch a call to the Triton server\n",
+    "    entry_script=\"score_densenet.py\",\n",
+    "    source_directory=os.path.join(\"source_dir\"),\n",
+    "    environment=env,\n",
+    ")\n",
+    "\n",
+    "aks_config = AksWebservice.deploy_configuration(\n",
+    "    cpu_cores=1, memory_gb=4, gpu_cores=1, compute_target_name=\"aks-gpu-deploy\"\n",
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:169]: I thought the compute target was created already? regardless we can create one in the test workspace so we can avoid creating one everytime in the notebook

File: notebooks/triton/aks-gpu-deploy.ipynb [hunk 9]
```diff
@@ -0,0 +1,268 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "!python test_service.py --endpoint_name=$aks_service_name"
```
Comment (lostmygithubaccount) [inline notebooks/triton/aks-gpu-deploy.ipynb:212]: same comment as above

**Review by lostmygithubaccount — CHANGES_REQUESTED** [review]
other general comments: 

- keep scripts in the code folder, following dslp convention(s) 
- not allowing a 32MB model in this repo, please download in the notebook from somewhere else 
- triton isn't a framework, shall we rename it to notebooks/onnx? 
-

File: README.md (hunk 1 above)
Comment (gvashishtha) [reply README.md:39]: is this a review for me or you?

Comment (lostmygithubaccount) [reply README.md:39]: this one's for me...although if you want to fix it go ahead :)

**Comment by gvashishtha** [comment]
@lostmygithubaccount ready for another review, please

**Pull request closed without merging**